packages/async-rewriter/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../config/tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./lib",
-    "allowJs": true
+    "allowJs": true,
+    "strict": false
```

Comment:
I’ve skipped the conversion for `async-rewriter` because it was particularly difficult to work with :confused: